src/main/java/bowling/domain/controller/BowlingGameController.java

```diff
@@ -0,0 +1,17 @@
+package bowling.domain.controller;
```

Comment:
비즈니스 문제를 해결하는 클래스들이 `domain` 패키지에 위치하면 어떨까요? 아래의 글을 참고하세요.
https://edu.nextstep.camp/s/qnaNJSHq/ls/G4jIzsLk

```suggestion
package bowling.controller;
```